### ISSUE TITLE
Unnecessary bloat please change your loader to this

### ISSUE THREAD

**Issue by casualdegenerate** [wendigoWare/1111111#1]
https://github.com/wendigoWare/1111111/blob/main/wendigoware#L56
There is a bunch of bloat at the start of the script.
You can remove a lot of it by doing these steps
1. Import your GUI into studio
2. Save the GUI as a roblox asset: right click the GUI, and clicking "Save to File..." make sure it's a rbxm file
3. Move the file under your Xploits' workspace
4. Write the following code to load your GUI into the game.
```lua
-- This start has an unloader, so you can reload your script(just make sure to put your connections into the table created in the getgenv() globals)
if GUI then -- Destroys the object if it exist already(so you can remove it)
  GUI:Destroy()
end
if connections then -- This will disconnect any connections you made throughout the script, so you can reopen the UI without any bugs
  for _, v in next, connections do
    v:Disconnect()
  end
end
getgenv().connections = {}
local fname = 'file of your gui.rbxm'
local mirror = 'https:// of your rbxm file so people can download it/file of your gui.rbxm' -- This has to be a rbxm file, not a url to the file's download
if not isfile(fname) then -- You will download the file only once(if you did not delete it) to prevent wasting bandwidth with 3000 lines of bullshit
  print('file not found, downloading from mirror') -- This is debug info, you can remove the print, and let it just write the file
  writefile(fname, game:HttpGet(mirror))
end
local asset = getsynasset(fname) --This will give you an ID to use now for Roblox.

getgenv().GUI = game:GetObjects(asset)[1] -- This will get the first object that was saved into the file, being the single GUIInstance you saved.

GUI.Parent = WhereEverYourHeartDesires -- Please remember to protect your GUIs

table.insert(connections, GUI.ChildAdded:Connect(function(child)-- We will now put our RBLXSignals inside of the connections table, so we can restart the script whenever we run it again. 
  print(child) -- This is just an example, you can replace this with your code, and connecitons, just follow this if you'd like it to remove it's self if you run the code again. 
end))
--You can treat the GUI instance as your GUI object now, so have fun with a loader that is only 20~ lines :)
```
Any questions please ask, and I will message you back in this issue, I have nothing else to do(as you can tell) and I'm open to answer anything that confuses you. 
TLDR; Use getsynasset instead of a GUI API to paste over 3000 lines of code into your script.

**Comment by wendigoWare** [comment]
> 

i dont remember making this but..thanks i guess
